dorpsgek_github/__main__.py

```diff
@@ -0,0 +1,45 @@
+import logging
+import http
+import sys
+
+from aiohttp import web
+
+from dorpsgek_github import github
+from dorpsgek_github.config import GITHUB_APP_PORT
+
+# List of modules that are enabled
+from dorpsgek_github.modules import (  # noqa
```

Comment:
why is this `#noqa` necessary?

Reply:
`flake8` otherwise rightfully says the imports are not used. He doesn't know there is a registry inside those files that are now kicked into working. So we know the imports are not used, and we are fine with that :) so we mute `flake8` from complaining